src/scripts.js

```diff
@@ -1,11 +1,103 @@
-// This is the JavaScript entry file - your code begins here
-// Do not delete or rename this file ********
-
-// An example of how you tell webpack to use a CSS (SCSS) file
+//CSS (SCSS) file
 import './css/base.scss';
 
 // An example of how you tell webpack to use an image (also need to link to it in the index.html)
-import './images/turing-logo.png'
+import './images/turing-logo.png';
+import './images/TravelTracker.png';
+
+import Trips from './Trips';
+import Traveler from './Traveler';
+import TravelersRepo from './TravelersRepo';
+
+import {getAllTravelersData, getAllTripsData, getAllDestinationsData} from './apiCalls';
+
+//global variables
+let currentTraveler;
+let tripsRepo;
+let allTravelerData = [];
+let allTripData = [];
+let allDestinationData = [];
+
+//querySelectors
+
+const yearExpenses = document.getElementById('yearly-expenses');
+const upcomingContainer = document.getElementById('upcoming');
+const pastContainer = document.getElementById('past');
+
+//event listeners
+
+window.addEventListener('load', fetchData)
+
+function fetchData() {
+  Promise.all([getAllTravelersData(), getAllTripsData(), getAllDestinationsData()])
+  .then(values => parseValues(values))
+}
+
+function parseValues(data) {
```

Comment:
i like the way you brought in your fetch calls!  never thought to push that data to an array 💯